fineract-provider/src/main/java/org/apache/fineract/interoperation/util/InteropUtil.java

```diff
@@ -26,7 +26,7 @@ private InteropUtil() {
 
     }
 
-    public static final String ISO8601_DATE_TIME_FORMAT = "yyyy-MM-ddTHH:mm:ss.SSS[-HH:MM]";
+    public static final String ISO8601_DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS";
```

Comment:
Why did you change this? It looks breaking change!

Reply:
but it cannot be parsed with SMART resolvement

Reply:
you mean STRICT?

Reply:
yes, sorry

Reply:
It's under discussion wether or not we can change it.

Reply:
the pattern was incorrect, has been fixed